aspnetcore/signalr/version-differences.md

```diff
@@ -86,6 +98,10 @@ npm install @aspnet/signalr
 
 The dependency on jQuery has been removed, however projects can still use jQuery.
 
+### Internet Explorer support
+
+Microsoft Internet Explorer 11 is the lowest supported version of IE in ASP.NET Core SignalR, versus IE 8 in ASP.NET SignalR.
```

Comment:
```suggestion
ASP.NET Core SignalR requires at least Microsoft Internet Explorer **11**.
```

"versus" feels awkward

Reply:
Eliminating "versus" is fine, but the suggested change eliminates the info about the corresponding version for ASP.NET 4.x SignalR.